tests/cupyx_tests/scipy_tests/linalg_tests/test_decomp_lu.py

```diff
@@ -5,8 +5,17 @@
 import cupy
 from cupy import testing
 import cupyx.scipy.linalg
-if cupyx.scipy._scipy_available:
-    import scipy.linalg
+try:
+    import scipy.linalg as scipy_linalg
+except ImportError:
+    scipy_linalg = None
```

Comment:
Please keep `scipy.linalg`.  In CuPy's tests, `scipy` or `cupyx.scipy` is often provided by `testing.numpy_cupy_*` and `import scipy.linalg` is required just because `import scipy` does not import the submodule.
```suggestion
try:
    import scipy.linalg
except ImportError:
    pass
```